README.md

```diff
@@ -173,4 +178,4 @@ Tap on **Clear Identity** button to clear the current user's session.
 ### Push Notifications & Deep link
 To receive a push notification, identify (create) a user with word 'ami' in the first name. Note that you need to be a [customer.io](https://customer.io/) developer/tester to receive a notification. 
 
-On tapping the push notification, you will be taken to the second screen i.e. to identify a user.
\ No newline at end of file
+On tapping the push notification, you will be taken to the second screen i.e. to identify a user.
```

Comment:
should this be `On tapping the push notification button`?

Reply:
Push notification here is referred to the notification received on the device for ami app. There is no push notification button in the application.